doozer/doozerlib/cli/release_gen_payload.py

```diff
@@ -603,15 +607,20 @@ def generate_payload_entries(self, assembly_inspector: AssemblyInspector) -> (Di
                 continue
             # No adjustment for private or public; the assembly's canonical payload content is the same.
 
-            entries: Dict[str, PayloadEntry]  # Key of this dict is release payload tag name
-            issues: List[AssemblyIssue]
-            entries, issues = PayloadGenerator.find_payload_entries(assembly_inspector, arch, self.full_component_repo(repo_type=RepositoryType.PUBLIC))
+            # entries: Dict[str, PayloadEntry]  # Key of this dict is release payload tag name
+            # issues: List[AssemblyIssue]
+            public_entries, public_payload_issues = PayloadGenerator.find_payload_entries(assembly_inspector, arch, self.full_component_repo(repo_type=RepositoryType.PUBLIC))
+            entries_for_arch[arch] = public_entries
+            self.assembly_issues.extend(public_payload_issues)
 
-            # Exclude issues for now, for private repo
-            private_entries, _ = PayloadGenerator.find_payload_entries(assembly_inspector, arch, self.full_component_repo(repo_type=RepositoryType.PRIVATE))
-            entries_for_arch[arch] = entries
+            # Always make sure that embargoed builds do not flow to public image stream
+            # If releasing after embargo lift, this can be permitted using 'EMBARGOED_CONTENT' code
+            embargo_issues = PayloadGenerator.embargo_issues_for_payload(public_entries)
+            self.assembly_issues.extend(embargo_issues)
+
+            private_entries, private_payload_issues = PayloadGenerator.find_payload_entries(assembly_inspector, arch, self.full_component_repo(repo_type=RepositoryType.PRIVATE))
             private_entries_for_arch[arch] = private_entries
-            self.assembly_issues.extend(issues)
+            self.assembly_issues.extend(private_payload_issues)
```

Comment:
i think we do not want to add these into the assembly issues list because it will just be the same issues as public, so we'll get duplicates (that's my memory looking at the logic this is replacing; and looks like issues only come from RHCOS at this point). the assembly issues, in addition to being checked for permits, are also added to annotations on the imagestream, so we don't want to bloat them any further.